### PR TITLE
Add WebAssembly build job to CI workflow

### PR DESCRIPTION
**Pull request by netolcc06**

### CODE REVIEW

File: .github/actions/install-dependencies/action.yml [hunk 1]
```diff
@@ -61,6 +65,21 @@ runs:
...
+      shell: bash
+      run: |
+        echo "::group::Install software for WebAssembly compilation"
+        sudo apt-get update
+        git clone https://github.com/emscripten-core/emsdk.git
+        cd emsdk/
+        ./emsdk install '3.1.18'
+        ./emsdk activate '3.1.18'
+        echo "/home/runner/work/cvc5/cvc5/emsdk" >> $GITHUB_PATH
+        echo "/home/runner/work/cvc5/cvc5/emsdk/upstream/emscripten" >> $GITHUB_PATH
```
Comment (daniel-larraz) [inline .github/actions/install-dependencies/action.yml:81]: One suggestion to make the specified paths more resilient to changes:
```suggestion
        echo "$(pwd)" >> $GITHUB_PATH
        echo "$(pwd)/upstream/emscripten" >> $GITHUB_PATH
```

File: INSTALL.rst [hunk 2]
```diff
@@ -95,7 +95,16 @@ code).
 which modifies how the wasm and glue code are built and how they behave. An ``-s``
 should precede each flag.
 
-For example, to generate modularized glue code, use:
```
Comment (daniel-larraz) [inline INSTALL.rst:98]: I would introduce the first example with:
`For example, to generate a HTML page, use:`

File: INSTALL.rst [hunk 3]
```diff
@@ -95,7 +95,16 @@ code).
...
+.. code:: bash
+
+  ./configure.sh --static --static-binary --auto-download --wasm=HTML --name=prod
+
+  cd prod
+  make            # use -jN for parallel build with N threads
+
+After that, you can run python -m http.server within prod/bin, open http://0.0.0.0:8000/cvc5.html with Chrome to visualize the page generated by Emscripten, write down a valid SMTLIB input, and press ESC twice to obtain its output.
```
Comment (daniel-larraz) [inline INSTALL.rst:105]: Minor formatting suggestions:
```suggestion
After that, you can run ``python -m http.server`` within ``prod/bin``, open http://0.0.0.0:8000/cvc5.html with Chrome to visualize the page generated by Emscripten, write down a valid SMTLIB input, and press ESC twice to obtain its output.
```

**Review by daniel-larraz — CHANGES_REQUESTED** [review]
I left some minor remarks.

**Review by daniel-larraz — APPROVED** [review]

File: INSTALL.rst [hunk 4]
```diff
@@ -95,7 +95,18 @@ code).
...
+For example, to generate a HTML page, use:
+
+.. code:: bash
+
+  ./configure.sh --static --static-binary --auto-download --wasm=HTML --name=prod
+
+  cd prod
+  make            # use -jN for parallel build with N threads
+
+After that, you can run ``python -m http.server`` within ``prod/bin``, open http://0.0.0.0:8000/cvc5.html with Chrome to visualize the page generated by Emscripten, write down a valid SMTLIB input, and press ESC twice to obtain its output.
```
Comment (mpreiner) [inline INSTALL.rst:107]: Is this only working with Chrome or is it supposed to be working with any browser?

**Pull request merged**